Gems/XR/Code/Include/XR/XRSystem.h

```diff
@@ -70,6 +71,7 @@ namespace XR
         AZ::Matrix4x4 CreateProjectionOffset(float angleLeft, float angleRight, 
                                              float angleBottom, float angleTop, 
                                              float nearDist, float farDist) override;
+        AZ::RHI::XRRenderingInterface* GetRHIXRRenderingInterface() override;
```

Comment:
nit: GetRHIXRRenderingInterface() **const** override

Reply:
Since it's returning a non-const pointer it's more appropriate semantically for the function to not being const. The only caller to `GetRHIXRRenderingInterface` so far needs the pointer returned to be non-const, so I didn't bother to add the const version of the function returning the const pointer, but if that's something we can add in the future if necessary.